tools/gcp-org-hierarchy-viewer/gcpohv_cli.py

```diff
@@ -59,8 +59,14 @@ def setup_clients(credentials_file):
           Manager.
     """
     creds = setup_credentials(credentials_file)
-    crm_v1 = discovery.build('cloudresourcemanager', 'v1', credentials=creds)
-    crm_v2 = discovery.build('cloudresourcemanager', 'v2', credentials=creds)
+    crm_v1 = discovery.build('cloudresourcemanager',
+                             'v1',
+                             credentials=creds,
+                             static_discovery=True)
```

Comment:
Fyi, I'm not sure this does anything.

According to:
https://github.com/googleapis/google-api-python-client/blob/master/googleapiclient/discovery.py
`static_discovery` will default to `True` when `discoveryServiceUrl` is also not provided

So it was already being set to `True`.

Reply:
Yeah, upping the version is the real change.  This is just being explicit.